### PR TITLE
TeamRadarr.Radarr version 3.2.2.5080

### PR DESCRIPTION
**Pull request by alteredpark**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17762)

### CODE REVIEW

File: manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
+Installers:
+- Architecture: x64
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml:7]:
```suggestion
- Architecture: x86
```

File: manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.installer.yaml:5]:
```suggestion
PackageVersion: 3.2.2
```
This also shows as 3.2.2 🤦‍♂️
image

File: manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.2.2
```
image

File: manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
+PackageLocale: en-US
+Publisher: Team.Radarr
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.locale.en-US.yaml:7]:
```suggestion
Publisher: Team Radarr
```

File: manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2.5080
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2.5080/TeamRadarr.Radarr.yaml:5]:
```suggestion
PackageVersion: 3.2.2
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by mikeparkie** [comment]
@OfficialEsco yep, understood now, I know that for next time. I'll re-raise if the devs update the packages.

**Comment by OfficialEsco** [comment]
> @OfficialEsco yep, understood now, I know that for next time. I'll re-raise if the devs update the packages.

Does it matter that much tho? How often does it update under a version like 3.2.2?

**Comment by mikeparkie** [comment]
Actually, that's a good point and thinking about it they all have auto updaters, so the short version isn't to bad. I'll accept those suggestions now 👍

**Comment by OfficialEsco** [comment]
I would recommend you to do these changes in GitHub Desktop as you need to change the folder name

File: manifests/t/TeamRadarr/Radarr/3.2.2/TeamRadarr.Radarr.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: TeamRadarr.Radarr
+PackageVersion: 3.2.2
+PackageLocale: en-US
+PackageUrl: https://radarr.video/
+Publisher: TeamRadarr
```
Comment (OfficialEsco) [inline manifests/t/TeamRadarr/Radarr/3.2.2/TeamRadarr.Radarr.locale.en-US.yaml:6]:
```suggestion
Publisher: Team Radarr
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Ohhnoo so close 🤦‍♂️

**Review by OfficialEsco — APPROVED** [review]
🎉

**Pull request merged**